student-store-ui/src/components/CheckoutForm/CheckoutForm.jsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react"
+import "./CheckoutForm.css"
+
+export default function CheckoutForm(props) {
+    return (
+        
```

Comment:
make sure you return something (or null). If i'm scaffolding components out I tend to just return a div and text so I can see if they are rending, ex: 
```
return (
   <div>Checkout form</div>
   )
   ```